docs/docs/manual/running.md

```diff
@@ -290,6 +290,15 @@ JAVA_OPTIONS=-Drefine.data_dir=usr/lib/OpenRefineWorkspace
 </Tabs>
 
 
+##### Proxy Configuration
+
+To use a proxy add the following configurations for either HTTP Proxy or HTTPS Proxy in the relevant [modification file](#modifications-set-within-files) : 
+
+-Dhttps.proxyHost= \
+-Dhttp.proxyHost= \
+-Dhttp.proxyPort=
+  
```

Comment:
We have a section dedicated to Java variables, perhaps it would make sense to integrate these variables there?